src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -86,11 +86,13 @@ private function listMessages(SymfonyStyle $io, int $max)
             /** @var RedeliveryStamp|null $lastRedeliveryStamp */
             $lastRedeliveryStamp = $envelope->last(RedeliveryStamp::class);
 
+            $lastRedeliveryStampWithException = $this->getLastRedeliveryStampWithException($envelope);
+
             $rows[] = [
                 $this->getMessageId($envelope),
                 \get_class($envelope->getMessage()),
                 null === $lastRedeliveryStamp ? '' : $lastRedeliveryStamp->getRedeliveredAt()->format('Y-m-d H:i:s'),
```

Comment:
Same here - probably better to be consistent and use `$lastRedeliveryStampWithException`. In practice, this will show the original failure date and message, which is probably best (and then they can get more details to see the full history).

Reply:
Good point. I updated it.